app/models/video.py

```diff
@@ -1,4 +1,28 @@
 from app import db
+from app.models.rental import Rental
 
 class Video(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime)
+    total_inventory = db.Column(db.Integer)
+    #rentals = db.relationship('Rental', backref = 'video', lazy = True)
+    #customers = db.relationship("Customer", secondary="rental", back_populates="videos")
+
+    def to_video_object(self):
+        return {
+        "id": self.id,
+        "title": self.title,
+        "release_date": self.release_date,
+        "total_inventory": self.total_inventory
+    }
+
+    def videos_rental_query(self):
+        rental_query = Rental.query.filter_by(video_id = self.id)
```

Comment:
As with the equivalent method in customer, make sure to include the check for the rental still being checked out. This should only report the count of currently outstanding rentals against this video, not those which have already been checked in.